math/CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ Ref: https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.j
 ## [Unreleased]
 
 ### Features
+* [#18552](https://github.com/cosmos/cosmos-sdk/pull/18552) Add safe arithmetic operations for `math.Int`.
```

Comment:
Please augment this by adding ` that return an error in case of an overflow or any mishap`